test/test_prototype_transforms_functional.py

```diff
@@ -199,21 +199,38 @@ def resize_bounding_box():
             yield SampleInput(bounding_box, size=size, image_size=bounding_box.image_size)
 
 
-class TestKernelsCommon:
-    @pytest.mark.parametrize("functional_info", FUNCTIONAL_INFOS, ids=lambda functional_info: functional_info.name)
-    def test_scriptable(self, functional_info):
-        jit.script(functional_info.functional)
-
-    @pytest.mark.parametrize(
-        ("functional_info", "sample_input"),
-        [
-            pytest.param(functional_info, sample_input, id=f"{functional_info.name}-{idx}")
-            for functional_info in FUNCTIONAL_INFOS
-            for idx, sample_input in enumerate(functional_info.sample_inputs())
-        ],
-    )
-    def test_eager_vs_scripted(self, functional_info, sample_input):
-        eager = functional_info(sample_input)
-        scripted = jit.script(functional_info.functional)(*sample_input.args, **sample_input.kwargs)
-
-        torch.testing.assert_close(eager, scripted)
+@pytest.mark.parametrize(
+    "kernel",
+    [
+        pytest.param(kernel, id=name)
+        for name, kernel in F.__dict__.items()
+        if not name.startswith("_")
+        and callable(kernel)
+        and any(feature_type in name for feature_type in {"image", "segmentation_mask", "bounding_box", "label"})
+        and "pil" not in name
+        and (
+            name
+            not in {
+                "get_image_size",
+                "get_image_num_channels",
+            }
+        )
```

Comment:
This is only temporary. If we have full coverage for eager vs scripted tests we can simply reuse this instead of this abomination. Still, I've checked manually and for now this covers all kernels that we want to test and should be somewhat robust for future cases.